python/mxnet/numpy/random.py

```diff
@@ -99,7 +99,8 @@ def uniform(low=0.0, high=1.0, size=None, dtype=None, ctx=None, out=None):
         ``low`` and ``high`` are both scalars. Otherwise,
         ``np.broadcast(low, high).size`` samples are drawn.
     dtype : {'float16', 'float32', 'float64'}, optional
-        Data type of output samples. Default is 'float32'
+        Data type of output samples. Default is 'float32' or 'float64', which
+        denends on your current default dtype.
```

Comment:
probably say something like "when np_default_type=True" to clearly state which setting will result in which data type.